definitions/ext-snowflake_account/golden_metrics.yml

```diff
@@ -0,0 +1,58 @@
+warehouseCount:
+  title: "Virtual warehouses (count)"
+  query:
+    select: uniqueCount(NAME)
+    from: SnowflakeAccount
+    where: "metric_type = 'snowflake.billing'"
+  displayAsValue: true
+databaseCount:
+  title: "Databases (count)"
+  query:
+    select: uniqueCount(DATABASE_NAME)
+    from: SnowflakeAccount
+    where: "metric_type = 'snowflake.database_storage'"
+  displayAsValue: true
+creditsUsed:
+  title: "Credits Used"
+  query:
+    select: average(CREDITS_USED_AVERAGE)
+    from: SnowflakeAccount
+    where: "metric_type = 'snowflake.billing'"
+    facet: NAME
+storageUsage:
+  title: "Storage Usage"
+  query:
+    select: average(FAILSAFE_BYTES), average(STAGE_BYTES), average(STORAGE_BYTES)
+    from: SnowflakeAccount
+    where: "metric_type = 'snowflake.storage_usage'"
+pipeUsage:
+  title: "Snowpipe Usage"
+  query:
+    select: average(CREDITS_USED_AVERAGE), average(BYTES_INSERTED_AVERAGE), average(FILES_INSERTED_AVERAGE)
```

Comment:
this will not work with signals and goes a bit against the golden metric concept.

We have 1 metric per board. I would reduce all of the multiples one to 1